ovos_plugin_manager/audio_transformers.py

```diff
@@ -1,7 +1,8 @@
-from ovos_plugin_manager.utils import PluginTypes, PluginConfigTypes
-from ovos_plugin_manager.templates.transformers import AudioTransformer
 from ovos_utils.log import LOG
 
+from ovos_plugin_manager.templates.transformers import AudioTransformer, AudioLanguageDetector
+from ovos_plugin_manager.utils import PluginTypes, PluginConfigTypes
+
 
 def find_plugins(*args, **kwargs):
     # TODO: Deprecate in 0.1.0
```

Comment:
im trying to go with 0.1.0 everywhere to mark the transition to semver, as it is hard to follow if we are on 0.0.X

these are old leftovers that should have been removed sooner, just here for imports backwards compat, as said in logs
```
    LOG.warning("This reference is deprecated. "
                "Import from ovos_plugin_manager.utils directly")
 ```
 
 nothing should be importing from here, this only remains due to an abundance of caution, warning has been there for a while so i say we just delete it

Reply:
https://github.com/OpenVoiceOS/ovos-plugin-manager/pull/254/files

removed these methods, also updated all other deprecation logs to 1.0.0 instead of 0.1.0